osx_bundle/misc/bundle/launcher.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
I guess it would be a good idea to change this to /bin/bash then? Since we don't test with anything else.

Reply:
It's entirely up to you. In its current incarnation, the script works as POSIX-compliant, but it could be safer to just use bash.

Reply:
Yeah,can you change it while you're at it?